pg_search/src/schema/mod.rs

```diff
@@ -577,9 +577,7 @@ impl SearchIndexSchema {
             }
 
             let id: SearchFieldId = match &config {
-                SearchFieldConfig::Ctid => {
-                    builder.add_u64_field(name.as_ref(), INDEXED | STORED | FAST)
```

Comment:
oops!  Removing `STORED` here broke vacuum.

Our vacuum support (the `SearchIndex::delete()` function) reads the whole `TantivyDocument` from the index and then looks up the "ctid" field.  If it's not stored, then there's no "ctid" value and docs don't ever get vacuumed away.